action.yml

```diff
@@ -11,35 +11,61 @@ runs:
           source ${{ github.action_path }}/flag-alt-text.sh
 
           if [ ${{ github.event.comment }} ]; then
-            comment=$COMMENT
-            issue=${{ github.event.issue.html_url }}
-            comment_owner=${{ github.event.comment.user.login }}
-            is_pr=${{ github.event.issue.pull_request.url != '' }}
```

Comment:
I changed this `is_pr` variable to be `type`, and set to one of:

- `issue_description`
- `issue_comment`
- `pr_description`
- `pr_comment`
- `discussion_description`

I think this will what we're dealing with more clear.